src/main/java/edu/greenblitz/tobyDetermined/subsystems/swerve/inputs/SwerveModuleInputs.java

```diff
@@ -0,0 +1,23 @@
+package edu.greenblitz.tobyDetermined.subsystems.swerve.inputs;
+
+
+import org.littletonrobotics.junction.AutoLog;
```

Comment:
Again about the inputs file, and why its in a folder on its own

Reply:
its in its own folder to make more cleaner

Reply:
its not anymore